dotcom-rendering/src/amp/components/StickyAd.tsx

```diff
@@ -1,9 +1,7 @@
-import { ClassNames } from '@emotion/react';
 import { text, textSans } from '@guardian/source-foundations';
-import { regionClasses } from '../lib/region-classes';
-import { RegionalAdProps, RegionalAd } from './Ad';
+import { realTimeConfig, RegionalAdProps as StickyAdProps, Ad } from './Ad';
```

Comment:
I'm not sure I'm convinced that this is the best thing to do. I'm trying to avoid just using `RegionalAdProps` for `StickyAd `. It might be clearer to think of a new name for `RegionalAdProps` that encompasses both the regional and sticky use case. 🤔

Reply:
Yeah feels like we could narrow this a bit. Is there a base props that `RegionalAdProps` and `StickyAdProps` can both extend from?

Reply:
How about [this](https://github.com/guardian/dotcom-rendering/pull/3709/commits/1d4fab7e7cfd3d9ea09b7a75ae4467f048b06385#diff-e161fa864f5d2b3e59703455fad31b8932f8cfc0dde517adc47ee45413ba2b2aR123)?

Reply:
I think that makes a lot of sense 👍